packages/manager/modules/dbaas-logs/src/logs/translations/Messages_fr_FR.json

```diff
@@ -218,6 +218,14 @@
   "logs_home_account_title": "Modifier le nom",
   "logs_home_account_description": "Modifier le nom {{displayName}} affiché dans le menu de gauche ainsi que dans l'entête.",
   "logs_home_account_display_name_field": "Nom",
+  "logs_home_account_status": "Statut",
+  "logs_home_status_ENABLED": "Activé",
+  "logs_home_status_DISABLED": "Désactivé",
+  "logs_home_status_INIT": "En cours d'installation",
+  "logs_home_status_TO_CONFIG": "A configurer",
+  "logs_home_resiliate": "Résilier",
+  "logs_terminate_service_success": "Votre demande de résiliation de votre LDP {{serviceId}} a été prise en compte. Un e-mail contenant la procédure vous a été envoyé.",
```

Comment:
`votre LDP` doesn't make sense -> `votre compte LDP` or `votre service LDP` ?

Reply:
It is written this way in the issue

Reply:
you've already created the following key so what's the usage for that one ?
```
"autorenew_agora_terminate_service_success_DBAAS_LOGS": "Votre demande de résiliation de votre LDP a été prise en compte. Un e-mail contenant la procédure vous a été envoyé.",
```